python/lsst/pipe/tasks/postprocess.py

```diff
@@ -798,64 +800,51 @@ class TransformCatalogBaseConfig(pipeBase.PipelineTaskConfig,
 
 
 class TransformCatalogBaseTask(pipeBase.PipelineTask):
-    """Base class for transforming/standardizing a catalog
+    """Base class for transforming/standardizing a catalog by applying functors
+    that convert units and apply calibrations.
 
-    by applying functors that convert units and apply calibrations.
-    The purpose of this task is to perform a set of computations on
-    an input ``DeferredDatasetHandle`` or ``InMemoryDatasetHandle`` that holds
-    a ``DataFrame`` dataset (such as ``deepCoadd_obj``), and write the
-    results to a new dataset (which needs to be declared in an ``outputDataset``
+    The purpose of this task is to perform a set of computations on an input
+    ``DeferredDatasetHandle`` or ``InMemoryDatasetHandle`` that holds a
+    ``DataFrame`` dataset (such as ``deepCoadd_obj``), and write the results to
+    a new dataset (which needs to be declared in an ``outputDataset``
     attribute).
 
     The calculations to be performed are defined in a YAML file that specifies
-    a set of functors to be computed, provided as
-    a ``--functorFile`` config parameter.  An example of such a YAML file
-    is the following:
+    a set of functors to be computed, provided as a ``--functorFile`` config
+    parameter. An example of such a YAML file is the following:
 
         funcs:
-            psfMag:
-                functor: Mag
-                args:
-                    - base_PsfFlux
-                filt: HSC-G
-                dataset: meas
-            cmodel_magDiff:
-                functor: MagDiff
-                args:
-                    - modelfit_CModel
-                    - base_PsfFlux
-                filt: HSC-G
-            gauss_magDiff:
-                functor: MagDiff
-                args:
-                    - base_GaussianFlux
-                    - base_PsfFlux
-                filt: HSC-G
-            count:
+            sourceId:
```

Comment:
What was the motivation for this change? The previous example was out of date, or non-functional?

Reply:
The initial motivation was to replace the mis-named flags section and replace the MagDiff examples, as MagDiff was going away. On review, MagDiff remains, but the example edits here were preserved to showcase more typical examples.